src/pages/member/MemberProfile.js

```diff
@@ -439,8 +628,8 @@ function MemberProfile(props) {
                     </Card>
                 </div>
 
-                <div style={isMobile ? {paddingTop: 20} : {paddingTop: 40}}>
-                    <Card style={isMobile? { height: 535 , marginLeft: 0} : { height: 495, marginLeft: 40}}>
+                <div style={isMobile ? {paddingTop: 36} : {paddingTop: 40}}>
+                    <Card style={isMobile? { height: 535 , marginLeft: 0 } : { height: 495, marginLeft: 40}}>
```

Comment:
i think it would be ideal if all styling was sorted at the top of the file instead of doing it in line